pkg/operator/latencyprofilecontroller/controller_status.go

```diff
@@ -0,0 +1,51 @@
+package latencyprofilecontroller
+
+import (
+	"context"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	// set of reasons used for updating status
+	reasonLatencyProfileUpdated         = "ProfileUpdated"
+	reasonLatencyProfileUpdateTriggered = "ProfileUpdateTriggered"
+	reasonLatencyProfileEmpty           = "ProfileEmpty"
+
+	wlpPrefix = "WorkerLatencyProfile"
+)
+
+func (c *LatencyProfileController) updateStatus(ctx context.Context, reason, message string, progressing, complete bool) (bool, error) {
```

Comment:
I would move to `controller.go` file

Reply:
Moved the `updateStatus` func to controller.go.